examples/test_examples.py

```diff
@@ -59,6 +59,14 @@ def buildFileList(examples, files=None):
 
 darwin_opengl_reason = ("pyopenGL cannot find openGL library on big sur: "
                         "https://github.com/python/cpython/pull/21241")
+       
+darwin_pyside611_mpl_broken = (
+    platform.system() == "Darwin" and
+    Qt.VERSION_INFO.startswith("PySide6 6.1.1 ")
+)
+darwin_pyside611_mpl_reason = (
+    "Matplotlib causes segfaults with PySide 6.1.1 on OSX even without drawing operations."
```

Comment:
Note that it is not a segfault, just a informational log containing the string "error"